src/screens/TermsConditionsScreen.tsx

```diff
@@ -28,17 +28,30 @@ interface Props {
   language: string;
 }
 
-export class TermsConditionsScreen extends React.PureComponent<Props> {
+interface State {
+  showWarring: boolean;
+  height: number;
+  agreedToTermsAndConditions: boolean;
+  agreedToPrivacyPolicy: boolean;
+}
+
+const TERMS_AND_CONDITIONS_MARGIN_BOTTOM = 32;
```

Comment:
empty line between the `const` and `export class`?

Reply:
Albo, since this variable is used only in one place, and it's the class below, I'd say it should be the class member

Reply:
moved to class